packages/client/src/Components/GameIntro/GameIntro.tsx

```diff
@@ -12,11 +21,33 @@ const BUTTON_HEIGHT = 42;
 const TIMER_WIDTH = 51;
 const TIMER_HEIGHT = 112;
 
-export const GameIntro = ({ onStart }: GameIntroProps) => {
+export const GameIntro = ({ onStart, onStop }: GameIntroProps) => {
```

Comment:
можно лучше: реакт имеет встроенный тип, который упрощает описание пропсов

```suggestion
export const GameIntro: FC<GameIntroProps> = ({ onStart, onStop }) => {
```